cmd/commands/client/command_client.go

```diff
@@ -176,3 +177,20 @@ func (cmd *Command) Kill() error {
 
 	return nil
 }
+
+// TODO this function can be aligned with server function when client and server options will merge into
+func getNetworkDefinition(options CommandOptions) metadata.NetworkDefinition {
+	network := metadata.TestNetDefinition
+
+	switch {
+	case options.LocalNet:
```

Comment:
Why not to have `default` in switch statement

Reply:
you will need bigger scope variable anyway, so either it can be left empty and initialized in default: of switch or it can be initialized right away and let the switch to do modification if needed. I chose the second approach